Binary files Project-X_0.91.0.orig/ProjectX.jar and Project-X_0.91.0/ProjectX.jar differ
Only in Project-X_0.91.0: build
diff -ru Project-X_0.91.0.orig/build.sh Project-X_0.91.0/build.sh
--- Project-X_0.91.0.orig/build.sh	2007-01-23 18:16:06.000000000 +0100
+++ Project-X_0.91.0/build.sh	2012-12-05 09:47:17.502860392 +0100
@@ -21,6 +21,11 @@
 mkdir -p build
 javac -encoding "ISO-8859-1" -deprecation -O -g:none -classpath lib/commons-net-1.3.0.jar:lib/jakarta-oro-2.0.8.jar -d build @sources.lst
 cp ./resources/* ./build
+
+cp -a htmls ReadMe.txt build
+rm -f build/htmls/projectx_desc_de_pdf.zip
+cp -a ReleaseNotes_0.91.0.txt build/ReleaseNotes.txt
+
 jar cfvm ProjectX.jar MANIFEST.MF -C build .
 
 ######################################################
